test/build/bundle_test.js

```diff
@@ -48,6 +48,9 @@ suite('Bundler', () => {
     bundledStream.on('end', (data) => {
       resolve(files);
     });
+    bundledStream.on('error', (err) => {
```

Comment:
I've hit enough problems in my life with callback arguments not lining up that I default to being very explicit.